specification/search/data-plane/Azure.Search/preview/2019-05-06-preview/searchservice.json

```diff
@@ -5773,13 +5773,238 @@
+            },
+            "delimitedTextHeaders": {
+              "type": "string",
+              "required": false,
+              "description": "For CSV blobs, specifies a comma-delimited list of column headers, useful for mapping source fields to destination fields in an index."
+            },
+            "delimitedTextDelimiter": {
+              "type": "string",
+              "required": false,
+              "description": "For CSV blobs, specifies the end-of-line single-character delimiter for CSV files where each line starts a new document (for example, \"|\")."
+            },
+            "firstLineContainsHeaders": {
+              "type": "boolean",
+              "required": false,
+              "default": true,
+              "description": "For CSV blobs, indicates that the first (non-blank) line of each blob contains headers."
+            },
+            "documentRoot": {
+              "type": "string",
+              "required": false,
+              "description": "For JSON arrays, given a structured or semi-structured document, you can specify a path to the array using this property."
+            },
+            "dataToExtract": {
+              "type": "string",
+              "required": false,
+              "$ref": "#/definitions/DataToExtract"
+            },
+            "imageAction": {
+              "type": "string",
+              "required": false,
+              "$ref": "#/definitions/ImageAction"
+            },
+            "allowSkillsetToReadFileData": {
+              "type": "boolean",
+              "required": false,
+              "default": false,
+              "description": "If true, will create a path //document//file_data that is an object representing the original file data downloaded from your blob data source.  This allows you to pass the original file data to a custom skill for processing within the enrichment pipeline, or to the Document Extraction skill."
+            },
+            "pdfTextRotationAlgorithm": {
+              "type": "string",
+              "required": false,
+              "$ref": "#/definitions/PdfTextRotationAlgorithm"
+            },
+            "queryTimeout": {
+              "type": "string",
+              "required": false,
+              "description": "Increases the timeout beyond the 5-minute default for Azure SQL database data sources, specified in the format \"hh:mm:ss\"."
+            }
+          },
           "additionalProperties": {
             "type": "object"
           },
           "description": "A dictionary of indexer-specific configuration properties. Each name is the name of a specific property. Each value must be of a primitive type."
         }
       },
-      "description": "Represents parameters for indexer execution."
+      "description": "Represents parameters for indexer execution.",
+      "externalDocs": {
+        "url": "https://docs.microsoft.com/en-us/rest/api/searchservice/create-indexer#parameters"
```

Comment:
Please remove en-us from all doc URLs so that readers of the generated documentation can be redirected to localized pages.

Reply:
Fixed in commit 18b17f8ce55b9ca0c51bb93c7ed7a26c567fa8ed.